server.js

```diff
@@ -647,6 +647,12 @@ module.exports.initExpress = function() {
         require('./middlewares/studentAssessmentAccess'),
         require('./pages/studentAssessmentInstanceFile/studentAssessmentInstanceFile'),
     ]);
+    app.use('/pl/course_instance/:course_instance_id/assessment_instance/:assessment_instance_id/time_left', [
```

Comment:
All of the other timer code refers to this as remaining rather than left. I don't really mind either way, but we should be consistent.
```suggestion
    app.use('/pl/course_instance/:course_instance_id/assessment_instance/:assessment_instance_id/time_remaining', [
```